service_container.rst

```diff
@@ -361,7 +361,7 @@ made. To do that, you create a new class::
                 );
             $this->mailer->send($message);
 
-            return $message;
+            return $happyMessage;
```

Comment:
Not sure this method should return anything. The controller should use a custom alert instead of this returned value.

Reply:
I agree with you that this is not an ideal example. I have merged this PR nonetheless as it would make the code at least work as it was supposed to be initially. We should handle the rest in a follow up PR.

Reply:
see #8339